sql/moz-fx-data-shared-prod/monitoring_derived/zyte_rss_load_job_monitoring_v1/bigeye_custom_rules.sql

```diff
@@ -0,0 +1,19 @@
+-- Alert when zyte_cache failures exceed the BigQuery change table limit
+-- {'name': 'zyte_cache_failures_critical', 'alert_type': 'count', 'range': {'max': 1400}, 'collections': ['Home Newtab'], 'schedule': 'Default Schedule - 13:00 UTC'}
+SELECT
+  zyte_cache_failure AS value
+FROM
+  `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
+WHERE
+  submission_date = CURRENT_DATE()
+  AND zyte_cache_failure > 1400;
+
+-- Alert when rss_feed_items failures exceed the BigQuery change table limit
+-- {'name': 'rss_feed_items_failures_critical', 'alert_type': 'count', 'range': {'max': 1400}, 'collections': ['Home Newtab'], 'schedule': 'Default Schedule - 13:00 UTC'}
+SELECT
+  rss_feed_items_failure AS value
+FROM
+  `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`
+WHERE
+  submission_date = CURRENT_DATE()
+  AND rss_feed_items_failure > 1400;
```

Comment:
We're primarily interested in monitoring successful job counts. Could you replace `_failure` with `_success` in this file?